SeaPublicWebsite/Views/EnergyEfficiency/ActionPlan/ActionPlanWithSavedRecommendations.cshtml

```diff
@@ -141,7 +151,7 @@
                        new()
                        {
                            Classes = "sea-saved-recommendations-table-headers",
-                           Html = @<a class="govuk-link" href=@Url.Action("Recommendation_Get", new {id = (int) recommendation.Key, reference = Model.PropertyData.Reference, fromActionPlan = true})>
+                           Html = @<a class="govuk-link" href="@Model.UrlPrefix@Url.Action("Recommendation_Get", new {id = (int) recommendation.Key, reference = Model.PropertyData.Reference, fromActionPlan = true})">
```

Comment:
At first glance having the two statements next to each other with no space looks wrong to me. It works but it's not immediately clear what it's doing. Instead of `href="@foo@bar"` could we do something like `href="@{foo + bar}"`? I think that would be more understandable.